odo/tests/test_append.py

```diff
@@ -12,3 +12,16 @@ def test_append_list_to_set():
     s = set([1, 2, 3])
     append(s, [4, 5, 6])
     assert s == set([1, 2, 3, 4, 5, 6])
+
+
+def test_append_set_to_list():
+    s = set([3, 4, 5])
+    lst = [1, 2, 3]
+    append(lst, s)
+    assert lst == [1, 2, 3, 3, 4, 5]
```

Comment:
Hmmm, for a set of int, since the hash() is just going to take int value, and we are not resizing the set here, so it seems the order will hold -- except when the implementation for set hash change?

so to be immune of that, how about:

s = set([3, 4, 5])
lst = [1, 2, 3]
append(lst, s)
assert sorted(lst) == [1, 2, 3, 3, 4, 5]